pkg/deletion/deletion.go

```diff
@@ -47,6 +47,26 @@ func ValidateAndConfirmDeletion(cmd *cobra.Command, args []string, checkExistenc
 	return nil
 }
 
+func ValidateAndConfirmDeletionCustomPluginVersion(cmd *cobra.Command, args []string, checkExistence func(string) bool, resourceType, pluginId, versionId string) error {
```

Comment:
Why do we need a specialized function here?

Reply:
because we want user get the info for both  pluginId and versionId while they want to delete the plugin version.
User can have many plugins and each plugin can have many versions.

Reply:
Hey @sgagniere, as of v4 aren't we just prompting the user with "y/n" for every deleted resource?

Reply:
Oops, missed this.

Yeah, all of our delete confirmation prompts are yes/no now.

Reply:
Hi @sgagniere , thanks for your reply! I created and merged a new [PR](https://github.com/confluentinc/cli/pull/2889) with y/n for deletion cmd.

Reply:
I think this branch was started before v4, so there will need to be a merge.